lms/djangoapps/courseware/views.py

```diff
@@ -102,6 +102,7 @@
 template_imports = {'urllib': urllib}
 
 CONTENT_DEPTH = 2
+REQUIREMENTS_DISPLAY_MODES = CourseMode.CREDIT_MODES + [CourseMode.VERIFIED]
```

Comment:
@awaisdar001 It is required that if the user is not taking the course as a credit mode then don't display the credit requirements on the progress page. For the credit course student can be in anyone of the following mode of enrollments.
`honor, audit, verified, credit`. Every student is enrolled as an `honor` mode initially. Then he/she can pursue the course either with `audit` mode or update to `verified`. Then the verified mode students are eligible for credit. Once the user has passed the course successfully then he can request for credit from the credit provider institutions. So the `verified` and `credit` modes are basically the `credit` track. The credit requirements are displayed to these modes only on the progress page.